modules/core/src/cuda/gpu_mat_nd.cu

```diff
@@ -0,0 +1,280 @@
+#include "opencv2/opencv_modules.hpp"
```

Comment:
Please use short license header: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

Reply:
I have added the license header.